packages/pluggable-widgets-tools/test-config/jest.native.config.js

```diff
@@ -3,33 +3,34 @@ const { join } = require("path");
 const projectDir = process.cwd();
 
 module.exports = {
-    preset: "react-native",
+    preset: hasDependency("@testing-library/react-native") ? "@testing-library/react-native" : "react-native",
+    testRunner: "jest-jasmine2",
     clearMocks: true,
+    haste: {
+        defaultPlatform: "android",
+        platforms: ["android", "ios", "native"],
+    },
     rootDir: join(projectDir, "src"),
     setupFilesAfterEnv: [
         join(__dirname, "test-index-native.js"),
-        ...(hasDependency("react-native-gesture-handler") ? ["react-native-gesture-handler/jestSetup.js"] : [])
+        ...(hasDependency("react-native-gesture-handler") ? ["react-native-gesture-handler/jestSetup.js"] : []),
+        ...(hasDependency("@testing-library/jest-native") ? ["@testing-library/jest-native/extend-expect"] : [])
     ],
     snapshotSerializers: ["enzyme-to-json/serializer"],
     testMatch: ["<rootDir>/**/*.spec.{js,jsx,ts,tsx}"],
-    transformIgnorePatterns: ["node_modules/(?!.*react-native)(?!victory-)"],
+    transformIgnorePatterns: ["node_modules/(?!(.*react-native.*|victory-)/)"],
     transform: {
-        "\\.tsx?$": [
-            "ts-jest",
-            {
-                isolatedModules: true,
-                tsconfig: { module: "commonjs" }
-            }
-        ],
-        "\\.jsx?$": join(__dirname, "transform-native.js")
+        "^.+\\.tsx?$": ["ts-jest", { tsconfig: { module: "commonjs" } }],
```

Comment:
The same here, it is rolling back the `isolatedModules: true,` ts-jest config.

Reply:
Im gonna check, some tests are failing and Im not sure why

Reply:
Restored